packages/@o3r/core/src/utils/deep-fill/deep-fill.ts

```diff
@@ -43,7 +43,7 @@ export function immutablePrimitive(obj: any, additionalMappers?: PrimitiveRevive
     return new Date(obj);
   } else if (obj instanceof Object) {
     // eslint-disable-next-line no-use-before-define
-    return deepFill(obj, obj, additionalMappers);
+    return obj === null ? null : deepFill(obj, obj, additionalMappers);
```

Comment:
can `obj` be null if it's `instanceof Object` ?

Reply:
![image](https://github.com/user-attachments/assets/59c30188-3801-47dd-84f9-e5b5ec39ffcd)